src/EnergyPlus/PlantCondLoopOperation.cc

```diff
@@ -3369,7 +3369,7 @@ namespace PlantCondLoopOperation {
                         SetupEMSActuator(ActuatorName,
                                          UniqueIDName,
                                          ActuatorType,
-                                         "[W]",
+                                         "[fraction]",
```

Comment:
@Myoldmopar @mitchute  Does this really work as a fraction? The other supervisory on/off actuators here use ` Units` which = "[on/off]".
```
EnergyManagementSystem:Actuator Available,CHILLED WATER LOOP,Supply Side Half Loop,On/Off Supervisory,[on/off]
EnergyManagementSystem:Actuator Available,CW DEMAND SIDE INLET PIPE,Plant Component Pipe:Adiabatic,On/Off Supervisory,[fraction]
```

Reply:
@mjwitte See this comment: https://github.com/NREL/EnergyPlus/issues/7026#issuecomment-434425235

Reply:
Well, fancy that. Who knew. Carry on.